mantaray/commands.py

```diff
@@ -154,6 +156,24 @@ def kick(view: View, core: IrcCore, nick: str, reason: str | None = None) -> Non
         else:
             view.add_message("You can use /kick only on a channel.", tag="error")
 
+    def away(view: View, core: IrcCore, away_message: str | None = None) -> None:
+        if away_message is None:
+            core.send("AWAY")
+            for view in view.server_view.get_subviews():
+                view.add_message(
+                    "*", ("You are no longer marked as being away", ["info"])
```

Comment:
I would prefer a separate command `/back` to be used for this. It's not very intuitive that typing `/away` makes you not be away. It's the complete opposite of what you'd expect it to do.